awx/api/fields.py

```diff
@@ -102,7 +102,12 @@ def __init__(self, **kwargs):
 
     def to_internal_value(self, pk):
         try:
+            pk = int(pk)
+        except (ValueError):
+            self.fail('invalid')
+        try:
+            pk = int(pk)
```

Comment:
just syntactic stuff, you converted it to an integer twice, so you don't need this 2nd one, and you don't want to add extra () around the exception class in these 2 places

Reply:
Updated!